docs/crash-reporting.md

```diff
@@ -12,6 +12,14 @@ Documentation for the specific libraries is included in the [Android Components
 
 The Glean crash ping format is documented [here](https://github.com/mozilla-mobile/android-components/blob/master/components/lib/crash/docs/metrics.md).
 
+## Breadcrumbs
+
+[Breadcrumbs](https://github.com/mozilla-mobile/android-components/blob/master/components/support/base/src/main/java/mozilla/components/support/base/crash/Breadcrumb.kt) are trail of events that are sent with each crash report to both Socorro and Sentry.  
```

Comment:
To make this documentation clearer, can you also include that the breadcrumb includes: